Sources/PostgresNIO/New/PostgresFrontendMessageEncoder.swift

```diff
@@ -167,6 +167,25 @@ struct PostgresFrontendMessageEncoder {
         self.buffer.writeMultipleIntegers(UInt32(8), Self.sslRequestCode)
     }
 
+    mutating func copyData(data: ByteBuffer) {
+        self.clearIfNeeded()
+        self.buffer.psqlWriteMultipleIntegers(id: .copyData, length: UInt32(data.readableBytes))
+        self.buffer.writeImmutableBuffer(data)
```

Comment:
I wonder if in this case, we should not copy the byteBuffer here. Instead we could forward the OG buffer, after we have sent the head. Would be two NIO write calls followed by a single flush. Would save at least one unnecessary copy call.

Reply:
Love that!